scripts/travis_test.sh

```diff
@@ -1,13 +1,80 @@
 #!/bin/bash
 RV=0
-cd examples/linux
+
+# Run all examples; this assumes PWD is examples/script
+run_examples() {
+    # concolic assumes presence of ../linux/simpleassert
+    HW=../linux/helloworld
+    SA=../linux/simpleassert
+    END_OF_MAIN=$(objdump -d $SA|awk -v RS= '/^[[:xdigit:]].*<main>/'|grep ret|tr  -d ' ' | awk -F: '{print "0x" $1}')
+    python ./concolic.py $END_OF_MAIN
+    if [ $? -ne 0 ]; then
+        return 1
+    fi
+
+    python ./count_instructions.py $HW |grep -q Executed
+    if [ $? -ne 0 ]; then
+        return 1
+    fi
+
+    gcc -static -g src/state_explore.c -o state_explore
+    ADDRESS=0x$(objdump -S state_explore | grep -A 1 '((value & 0xff) != 0)' |
+            tail -n 1 | sed 's|^\s*||g' | cut -f1 -d:)
+    python ./introduce_symbolic_bytes.py state_explore $ADDRESS
+    if [ $? -ne 0 ]; then
+        return 1
+    fi
+
+    gcc -x c -static -o hello - <<-EOF
+    #include <stdio.h>
+    int main(){return 0;}
+	EOF
+    python ./run_simple.py hello
+    if [ $? -ne 0 ]; then
+        return 1
+    fi
+
+    MAIN_ADDR=$(nm $HW|grep 'T main' | awk '{print "0x"$1}')
+    python ./run_hook.py $HW $MAIN_ADDR
+    if [ $? -ne 0 ]; then
+        return 1
+    fi
+
+    # Straight from the header of state_control.py
+    gcc -static -g src/state_explore.c -o state_explore
+    SE_ADDR=0x$(objdump -S state_explore | grep -A 1 'value == 0x41' |
+               tail -n 1 | sed 's|^\s*||g' | cut -f1 -d:)
+    python ./state_control.py state_explore $SE_ADDR
+    if [ $? -ne 0 ]; then
+        return 1
+    fi
+
+    return 0
+}
+
+pushd examples/linux
 if make; then
     echo "Successfully built Linux examples"
+    for example in $(make list); do
+        if ! ./$example < /dev/zero > /dev/null ; then
+            echo "Failed to run $example"
+            RV=1
+            break
+        fi
+    done
 else
     echo "Failed to build Linux examples"
     RV=1
 fi
-cd ../..
+popd
+
+if [ "$RV" -eq "0" ]; then
+    echo "Successfully ran Linux examples"
```

Comment:
can we make the output messages a little clearer as to whether the example binaries or the examples scripts were successful?